spire/templates/apps/dovetail-metrics-export.yml

```diff
@@ -257,11 +257,12 @@ Resources:
               "IntegrationsIDs": [1],
               "CompressionType": "NONE",
               "DestinationFormat": "CSV",
-              "ObjectPrefix": "boostr/",
+              "ObjectPrefix": "",
```

Comment:
I think it'd be better to not include `ObjectPrefix` than have it be an empty string